Multisig/UI/Dapps/DappsViewController.swift

```diff
@@ -270,3 +275,37 @@ extension DappsViewController: QRCodeScannerViewControllerDelegate {
         dismiss(animated: true, completion: nil)
     }
 }
+
+extension DappsViewController: WalletConnectKeysServerControllerDelegate {
+    func shouldStart(session: Session, completion: @escaping ([KeyInfo]) -> Void) {
```

Comment:
This should be called after the scanner returns a URL

Reply:
@biafra23 I would suggest to leave this functionality to the "Connect screen" I'm working on right now. For this issue, if you could trigger opening Settings and opening the "Connect To Web" screen after scanning the QR code, this would be good to go!

Reply:
Really to Connect to web? Then they have to scan again. I could go to the key selection screen. Or will you change this later?